src/pages/login/index.js

```diff
@@ -44,6 +49,15 @@ const Login = () => {
     return <Redirect to={routesPaths.index} />;
   }
 
+  const responseFacebook = response => {
+    console.log(response);
+    setData(response);
```

Comment:
With this function you are grabbing the Facebook API's response, but you aren't actually doing anything with it apart from saving it inside the state. If we consider the Target API's [documentation](https://targetmvd.docs.apiary.io/#reference/0/login-with-facebook/login-with-facebook) you should make and HTTP POST to the `/users/facebookRequest` endpoint with the `accessToken`. 
I tried this doing but the API's doesn't seem to work, if i were you, I'd speak to my leader/mentor asking if it's really necessary to make this feature work.